docs/managing-compute-resources.asciidoc

```diff
@@ -6,19 +6,19 @@ endif::[]
+To help the Kubernetes scheduler correctly place Pods in available Kubernetes nodes and ensure quality of service (QoS), it is recommended to specify the CPU and memory requirements for objects managed by the operator (Elasticsearch, Kibana or APM Server). In Kubernetes, `requests` defines the minimum amount of resources that must be available for a Pod to be scheduled; `limits` defines the maximum amount of resources that a Pod is allowed to consume. For more information about how Kubernetes uses these concepts, see: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/[Managing Compute Resources for Containers].
 
 [float]
 [id="{p}-compute-resources"]
 === Set compute resources
 
-Compute resource constraints can be set in the `podTemplate` of objects managed by the operator.
+You can set compute resource constraints in the `podTemplate` of objects managed by the operator.
 
 [float]
 [id="{p}-compute-resources-elasticsearch"]
 ==== Set compute resources for Elasticsearch
 
-For Elasticsearch objects, it is important to consider the heap size when setting resource requirements. The recommendation for heap size is that it should be half the size of RAM allocated to the pod. To minimize disruption caused by pod evictions due to resource contention, it is highly recommended to run Elasticsearch pods at the "Guaranteed" QoS level by setting both `requests` and `limits` to appropriate values. It is also worth bearing in mind that Kubernetes throttles containers exceeding the CPU limit defined in the `limits` section. Do not set this value too low or it would affect the performance of Elasticsearch even if you have enough resources available in the Kubernetes cluster.
+For Elasticsearch objects, make sure to consider the heap size when you set resource requirements. The heap size should be half the size of RAM allocated to the Pod. To minimize disruption caused by Pod evictions due to resource contention, you should run Elasticsearch pods at the "Guaranteed" QoS level by setting both `requests` and `limits` to appropriate values. Consider also that Kubernetes slows down containers that exceed the CPU limit defined in the `limits` section. Do not set this value too low or it would affect the performance of Elasticsearch even if you have enough resources available in the Kubernetes cluster.
```

Comment:
```suggestion
For Elasticsearch objects, make sure to consider the heap size when you set resource requirements. The heap size should be half the size of RAM allocated to the Pod. To minimize disruption caused by Pod evictions due to resource contention, you should run Elasticsearch pods at the "Guaranteed" QoS level by setting both `requests` and `limits` to the same value. Consider also that Kubernetes slows down containers that exceed the CPU limit defined in the `limits` section. Do not set this value too low or it would affect the performance of Elasticsearch even if you have enough resources available in the Kubernetes cluster.
```

I realize this was present in the original doc too but it looks like we were insufficiently precise before. It looks like the github diff isn't making it obvious, but this is just a change from "to appropriate values" to "to the same value"